### PR TITLE
A post about current LC activity

### PR DESCRIPTION
**Pull request by weaverbel**
I had to create this in the 03 folder as 04 wasn't yet created.

### CODE REVIEW

File: _posts/2017/03/2017-04-06-libcarp-update.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+feedback on what archivists want through this 
+[form](https://docs.google.com/forms/d/e/1FAIpQLSdc9RHNmgffnt9UtLfgIhS_MGUzh0wh-HQNX24wBz5c4mcf1g/viewform).
+
+Here is [more information on what we hope to tackle in the sprint](https://github.com/LibraryCarpentry/librarycarpentry.github.io/issues/23).  
+
+So far, there have been about 15 Library Carpentry workshops since last year's global sprint 
+provided the impetus for this new community. Our [chat room](https://gitter.im/weaverbel/LibraryCarpentry) is very 
+active and issues are constantly debated on GitHub. 
+
+So far, Australia, Canada, the US, the Netherlands, the UK and South Africa are the main places
```
Comment (gvwilson) [inline _posts/2017/03/2017-04-06-libcarp-update.md]: Repeat of "So far" from previous para.

File: _posts/2017/03/2017-04-06-libcarp-update.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+So far, Australia, Canada, the US, the Netherlands, the UK and South Africa are the main places 
+for activity with New Zealand about to join the bandwagon. 
+You can follow us on [Twitter](https://twitter.com/LibCarpentry).
+
+I have been invited by the Australian Society of Archivists to teach a Library Carpentry workshop 
+as a tie-in to their annual conference in September. 
+
+There is an upcoming workshop in South Africa in May, as well as plans for one in Ottawa a bit later on in the year. 
+Hopefully there will be many more workshops in the US after the Portland instructor training mints 30 new instructors.
```
Comment (gvwilson) [inline _posts/2017/03/2017-04-06-libcarp-update.md]: Perhaps close with a para on "If you would like to get involved..."

**Review by gvwilson — COMMENTED** [review]
Busy year :-)

**Comment by jduckles** [comment]
@weaverbel For future reference, when you're the first poster of the month, you can create the 04 directory and push it. Won't cause any problems.

**Comment by weaverbel** [comment]
Ok @gvwilson @jduckles Fixed and new version there now.

**Pull request merged**